webapp/src/ts/services/debug.service.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Manage debug mode.
+ *
+ * When enabled:
+ *
+ *   - persist setting to a cookie
+ *   - put PouchDB in debug mode
+ *
+ * Note: When debugging, make sure to enable the logs by selecting 'verbose' and 'info' options
```

Comment:
Before we used to have `$logProvider.debugEnabled(bool);` which is only available in AngularJS. 
I did some research and test, and turns out that the new Angular doesn't provide something similar. It do have an "enableDebugTool" but this is used for profiling components.
Besides, we're relying on the native console.log/debug/warn/error for logging, I think we don't need to enable anything extra, so I just put a comment noting that we need to have the right filters in the console from dev tools to be able to see the right logs 🤷‍♀️